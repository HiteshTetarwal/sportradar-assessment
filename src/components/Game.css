.game-container {
    display: flex;
    flex-direction: column;
    align-items: center;
    border: 1px solid black;
    padding: 10px;
    margin-bottom: 10px;
}

.game-header {
    font-size: 24px;
    font-weight: bold;
    margin: 0;
}

.game-scores {
    font-size: 18px;
    margin: 0;
}