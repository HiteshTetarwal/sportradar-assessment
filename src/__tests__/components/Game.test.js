import React from "react";
import renderer from "react-test-renderer";
import Game from "../../components/Game";

test("Game component renders correctly", () => {
  const component = renderer.create(
    <Game homeTeam="Spain" awayTeam="Brazil" homeScore={0} awayScore={0} />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
