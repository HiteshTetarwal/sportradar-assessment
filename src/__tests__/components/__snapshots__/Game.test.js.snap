// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game component renders correctly 1`] = `
<div
  className="game-container"
>
  <h2
    className="game-header"
  >
    Spain vs. Brazil 
  </h2>
  <p
    className="game-scores"
  >
    0 - 0
  </p>
</div>
`;
